### PR TITLE
FFmpeg foss2016a

### PR DESCRIPTION
**Pull request by RvDijk**
FFmpeg with foss2016a

### CODE REVIEW

**Comment by boegel** [comment]
Jenkins: ok to test

File: easybuild/easyconfigs/f/FFmpeg/FFmpeg-2.8.7-foss-2016a.eb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+sources = [SOURCELOWER_TAR_BZ2]
+source_urls = ['http://ffmpeg.org/releases/']
+
+dependencies = [
+    ('NASM', '2.11.08'),
+    ('zlib', '1.2.8'),
+    ('x264', '20160114'),
```
Comment (boegel) [inline easybuild/easyconfigs/f/FFmpeg/FFmpeg-2.8.7-foss-2016a.eb]: existing easyconfig uses `NASM` 2.12.01 and `x264` 2016114, so bump to those versions here too?

Comment (boegel) [inline easybuild/easyconfigs/f/FFmpeg/FFmpeg-2.8.7-foss-2016a.eb]: (sorry, I meant `x264` 20160430)

**Comment by RvDijk** [comment]
depends on #3019 and #3020

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2159.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/dcb13d3764230c4a38bc582a981793d9 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2417.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/32bbf5dc815d88a003d81d1fb255c4c4 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @RvDijk!

**Pull request merged**